src/repl.ml

```diff
@@ -0,0 +1,96 @@
+open Import
+
+let name = "REPL"
+
+let t = ref None
```

Comment:
Is there a way to make this part of the instance state?

Reply:
Good idea, done